src/Workspaces/Core/Portable/CodeGeneration/Symbols/CodeGenerationMethodSymbol.cs

```diff
@@ -99,6 +99,8 @@ public override ImmutableArray<ITypeSymbol> TypeArguments
         public override bool IsReadOnly => Modifiers.IsReadOnly;
         public override bool IsInitOnly { get; }
 
+        public override System.Reflection.MethodImplAttributes ImplementationAttributes => default;
```

Comment:
This assumes `MethodImplAttributes.IL`. If this is wrong, it could be related to my comment https://github.com/dotnet/roslyn/pull/50560/files#r559016146